src/Sulu/Bundle/AdminBundle/Resources/js/package.json

```diff
@@ -27,6 +28,7 @@
         "enzyme-to-json": "^1.5.1",
         "identity-obj-proxy": "^3.0.0",
         "jest": "^20.0.4",
+        "pretty": "^2.0.0",
```

Comment:
`pretty` is just a dev-dependency which beautifies a string of html. This is used for testing the html of the overlay which is ported to the body.